modules/ossm-cert-manager-integration.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-security.adoc
+
+:_content-type: CONCEPT
+[id="ossm-cert-manager-integration_{context}"]
+== Integrating cert-manager with Service Mesh
+
+cert-manager adds certificates and certificate issuers as resource types in Kubernetes clusters, and simplifies the process of obtaining, renewing and using those certificates. 
+
+It obtains certificates from a variety of Issuers, both popular public Issuers as well as private Issuers, and ensures the certificates are valid and up-to-date. It also attempts to renew certificates at a configured time before expiry.
```

Comment:
I would also add a note that cert-manager enables Istio users to integrate with their private or external PKI.